integration/spark-common/src/main/scala/org/apache/spark/util/CarbonReflectionUtils.scala

```diff
@@ -140,10 +140,15 @@ object CarbonReflectionUtils {
 
   def getSessionState(sparkContext: SparkContext, carbonSession: Object): Any = {
     if (sparkContext.version.startsWith("2.1")) {
-      createObject("org.apache.spark.sql.hive.CarbonSessionState", carbonSession)._1
+      val className = sparkContext.conf
+        .get("spark.carbon.sessionstate.classname", "org.apache.spark.sql.hive.CarbonSessionState")
```

Comment:
make these string in CarbonCommonConstant instead of hard coding it here

Reply:
makespark.carbon.sessionstate.classname string in CarbonCommonConstant instead of hard coding it here. --- ok

and change org.apache.spark.sql.hive.CarbonSessionState to CarbonSessionState.getClass.getName --  CarbonSessionState belongs to spark2 module so we cannot access from spark-common module